supertokens_python/recipe/session/framework/flask/__init__.py

```diff
@@ -35,15 +36,18 @@ def verify_session(
     ] = None,
     user_context: Union[None, Dict[str, Any]] = None,
 ) -> Callable[[_T], _T]:
-    if user_context is None:
-        user_context = {}
+    _ = user_context
```

Comment:
why is this needed?

Reply:
Usage of user_context is 2nd order nested and somehow linter say it's not used. So they throw an error.